schema/schema.py

```diff
@@ -136,7 +139,8 @@ def format_param(param: click.core.Option) -> SamCliParameterSchema:
         formatted_param_type = param_type or "string"
 
     formatted_param: SamCliParameterSchema = SamCliParameterSchema(
-        param.name or "", formatted_param_type, clean_text(param.help or "")
+        param.name or "", formatted_param_type, clean_text(param.help or ""),
+        items="string" if formatted_param_type == "array" else None
```

Comment:
will it be always an array of Strings ?

Reply:
Yes. From looking through the parameters, it seems like all parameters that can be lists (only `notification_arns` and `capabilities`) have their items as strings.

Reply:
I will not block the PR on this, but we should think on a away to make it generic, as in future we can add new parameters that accept an array of different types than Strings.

Reply:
Yup, will look into investigating click a little more to see if we can pull those data types dynamically 👍